### DISCUSSION TITLE
conflicts with working title mods (G1/3000?)

### DISCUSSION THREAD

**Discussion by andrejetcat** [flybywiresim/aircraft#4613] [Q&A] [answered]
I just want to know, if the working title mods (I have the G1000/3000 mod + TBM improvement mod) can cause problems with the A32NX ? somebody know, if there is anything reported or known? 

Thanks a lot and thanks for this AWESOME project!!!

**Comment by hotshotp** [comment, marked as answer]
We are hard forked, no more mod conflicts
